StripeConnect/StripeConnect/Source/Internal/StripeConnectConstants.swift

```diff
@@ -12,7 +12,6 @@ enum StripeConnectConstants {
     /**'
      Pages or navigation requests matching any of these hosts will...
      - Automatically grant camera permissions
-     - Accept downloads (TODO MXMOBILE-2485)
```

Comment:
We're not using allowedHosts for downloads because most of our downloads come from external hosts (e.g. S3)